www/build.sh

```diff
@@ -45,7 +45,7 @@ rustc --version
 # is set up to serve them.
 export ROC_DOCS_URL_ROOT=/builtins
 
-cargo run --bin roc-docs crates/compiler/builtins/roc/*.roc
+cargo run --release --bin roc-docs crates/compiler/builtins/roc/*.roc
```

Comment:
I expect this to be better for caching, because all earlier cargo invocations were done with `--release`.